utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -111,6 +110,37 @@ const val LEARNER_STUDY_ANALYTICS = "learner_study_analytics"
  */
 const val LEARNER_STUDY_ANALYTICS_DEFAULT_VALUE = false
 
+/**
+ * Qualifier for a feature flag that controls whether learners may be allowed (via an
+ * admin-controlled setting) to use a special in-lesson button for quickly switching between content
+ * languages.
+ *
+ * This is generally expected to only be used in tandem with [EnableLearnerStudyAnalytics].
+ */
+@Qualifier annotation class EnableFastInLessonLanguageSwitching
+
+/** The platform parameter name corresponding to [EnableFastInLessonLanguageSwitching]. */
+const val FAST_IN_LESSON_LANGUAGE_SWITCHING = "fast_in_lesson_language_switching"
```

Comment:
fast_language_switching_in_lesson or fast_language_switching_in_lessons, perhaps?

Reply:
This has been adjusted per your earlier suggestion. It's now fast_language_switching_in_lesson.